src/api/GameController.js

```diff
@@ -192,22 +192,33 @@ exports.getRecentEventGames = function(req, res) {
     });
 };
 
-exports.getSuggestedPlayers = function(req, res) {
-  const max_players = req.query.max_players === undefined || 3;
+exports.getSuggestedPlayers = function (req, res) {
+  const max_players = req.query.max_players || 3;
```

Comment:
It was on purpose not to cancel a zero. But I did not think about the fact that 0 does not make sense in this context. max_players should be validated more strictly beforehand to return an error on zero.